### PR TITLE
edn-/property configuration symmetry

### PR DESCRIPTION
**Pull request by deas**
I expected edn-/property configuration meant to be symmetrical. This was not the case so far. In fact, it was not possible to load edn from a file. My expectations could be wrong. This PR makes it as symmetrical as I expected things. Other than that, I decided `:resource` may be a bit clearer than `:properties`, so I changed it along the way as well.

### CODE REVIEW

**Comment by chrisbetz** [comment]
Contributest Du da nicht in den falschen Fork ;)

Und was ist mal wieder mit Mittagessen (geht an Andreas und an Christian). Und gehört das überhaupt hier hin? Schöne Grüße, auch wenn der Kommentar nachher gelöscht ist ;)

File: src/de/otto/tesla/stateful/configuring.clj [hunk 1]
```diff
@@ -24,26 +24,28 @@
...
         (io/reader)
         (PushbackReader.)
         (read))))
 
 (defn load-config-from-edn-files []
-  (let [defaults (load-edn "default.edn")
-        application (load-edn (or (:config-file env/env) "application.edn"))
-        local (load-edn "local.edn")
+  (let [defaults (load-edn "default.edn" :resource)
+        application (load-edn (or (:config-file env/env) "application.edn") :file)
```
Comment (meandor) [inline src/de/otto/tesla/stateful/configuring.clj:47]: Why do you expect the `default.edn` and `local.edn` to be resources but the `application.edn` to be a file?

Comment (deas) [inline src/de/otto/tesla/stateful/configuring.clj:47]: I expected symmetry between property- and edn configuration in the first place. Resolving `default` and `local` from the classpath and `application` from file is what happens/ed when you use properties.

**Comment by stammi** [comment]
Hi @deas,

thanks a lot for spotting this. As this PR was already to old (my bad, sorry) I cherry-picked your commit. I then further changed the behaviour to be consistent between edn and properties and hopefully not break any existing application (see CHANGES.MD). Please let us know, if that does not work for you.

cheers, Stammi

**Pull request closed without merging**